MacOS brew ITK compilation error

Related to #531, but possibly seperate

STIR ITK (via HomeBrew) install issue. Setting a clean directory to build in allows for cmake configeration and generation (work around for #531's issue). However, the make fails on:
```
[  8%] Building CXX object src/IO/CMakeFiles/IO.dir/ITKImageInputFileFormat.cxx.o
/Users/roberttwyman/bin/STIR/STIR/src/IO/ITKImageInputFileFormat.cxx:513:69: error: no member named 'VECTOR' in 'itk::ImageIOBase'
      if (reader->GetImageIO()->GetPixelType() != itk::ImageIOBase::VECTOR) {
                                                  ~~~~~~~~~~~~~~~~~~^
1 error generated.
make[2]: *** [src/IO/CMakeFiles/IO.dir/ITKImageInputFileFormat.cxx.o] Error 1
make[1]: *** [src/IO/CMakeFiles/IO.dir/all] Error 2
make: *** [all] Error 2
```

--- a/src/IO/ITKImageInputFileFormat.cxx
+++ b/src/IO/ITKImageInputFileFormat.cxx
@@ -569,7 +569,13 @@ read_file_itk(const std::string &filename)
                 __FILE__, __LINE__);
           return NULL; }
 
-      if (reader->GetImageIO()->GetPixelType() != itk::ImageIOBase::VECTOR) {
+      if (reader->GetImageIO()->GetPixelType() != 
+#if ITK_VERSION_MAJOR<5 || (ITK_VERSION_MAJOR==5 && ITK_VERSION_MINOR==0)
+          itk::ImageIOBase::VECTOR
+#else
+          itk::IOPixelEnum::VECTOR
+#endif
+         ) {
           error("read_file_itk: Image type should be vector %s:%d.",
                 __FILE__, __LINE__);
           return NULL; }

